src/docs/issue-platform.mdx

```diff
@@ -0,0 +1,105 @@
+
+If operating from an external service then pass a json payload that matches the <Link to="#issue-occurrence-schema">Issue Occurrence Schema</Link> to the `ingest-occurrences` topic on the <Link to="https://github.com/getsentry/getsentry/blob/97039335a29517a7e48149148cdd7a30a60971e7/getsentry/conf/settings/prod.py#L567-L571">issue occurrences Kafka cluster</Link>
+
+### Issue Occurrence Schema:
+Required:
+- `id`: str. A user generated uuid in string format representing a unique id for this occurrence.
+Generally it's a good idea to generate this separately to your event id.
+- `project_id`: int. Id of the project this occurred in
+- `fingerprint`: Sequence[str].  Unique identifier for the root cause of this problem.
+Used for aggregating occurrences together into groups. We only make use of the first
+fingerprint for the moment - if you have a use case for additional fingerprints let the
+issue platform team know.
+- `issue_title`: str. A short description of the problem. This will be used as the title
+of the issue and also used in notifications.
+- `subtitle`: str: [todo]
+- `evidence_data`: Mapping[str, Any]. An arbitrary map of data. The issue platform doesn't
+make use of this data directly, it just make it available for use on the frontend and in
+notifications.  This can be used to customize the issue details ui, notifications and so on.
+- `evidence_display`: Sequence[<Link to="#issue-evidence-schema">IssueEvidence</Link>]. A sequence of `IssueEvidence` entries. These are used to display generic information about your issue in the issue details ui and in various notifications. Even if you are customizing the ui and email, it's recommended to include this so that we can display information correctly on all our other integrations. Typically, these will be displayed in a table when space allows. For space limited integrations we'll display one row that is labelled `important` if available.
+- `type`: int: The type of issue you are creating. Maps to `type_id` in your registered `GroupType`.
+- `detection_time`: float. A timestamp representing when the issue was detected.
+- `level`: str
+- `culprit`: str
+
+One of:
+- `event_id`: str. The event id of an existing event, typically this would be an error or transaction id.
+- `event`: <Link to="#event-schema">Event</Link>
+
+Optional:
+- `resource_id`: str | None. An identifier used to link to an external system to load data.
+For example, this could be an id that we use to build a link to a specific product page.
+
+### Issue Evidence Schema
```

Comment:
I'd be useful to link to the JSON schema validation you have. I always found them more useful to understand a schema than a list like this. Of course, we still need the description of the fields here.